server/server.go

```diff
@@ -342,31 +346,37 @@ func (srv *Server) finaliseConn(ctx context.Context, conn session.Conn, l Listen
 func (srv *Server) defaultGameData() minecraft.GameData {
 	return minecraft.GameData{
 		// We set these IDs to 1, because that's how the session will treat them.
-		EntityUniqueID:               1,
-		EntityRuntimeID:              1,
-		Difficulty:                   2,
```

Comment:
Any reason to remove the difficulty normal?

Reply:
No particular reason, readded.